doc/glossary/index.rst

```diff
@@ -31,3 +31,374 @@ Glossary
+           role.
+
+        .. note:: Use this carefully. Imagine you defined a resolver to a specific
+           group in your Active Directory to be the pricacyIDEA admins. Then the Active
+           Directory domain admins can
+           simply add users to be administrator in privacyIDEA.
+
+        You define the administrative realms in the config file ``pi.cfg``, which is
+        usually located at ``/etc/privacyidea/pi.cfg``::
+
+           SUPERUSER_REALM = ["adminrealm1", "super", "boss"]
+
+        In this case all the users in the realms "adminrealm1", "super" and "boss"
+        will have administrative rights in the WebUI, when they login with this realm.
+
+        As for all other users, you can use the :ref:`policy_login_mode` to define,
+        if these administrators should login to the WebUI with their userstore password
+        or with an OTP token.
+
+   Userstore
+        Are the locations, where the users are managed. This can be LDAP directories
+        or especially Active Directory, some times FreeIPA or the Redhat 389 service.
+        But classically users are also located in files like /etc/passwd on standalone unix systems.
+        Web services often use SQL databases as user store.
+
+   Userinfo
+        These are the user attributes as they are determined by the respective resolver.
+        This is configured via the attribute mappings of resolvers (see :ref:`useridresolvers`).
+
+   Challenge
+   Multi Challenge
+        If a user wants to authenticate with his username and password, privacyIDEA will check whether an active
+        challenge response token exists for this user. In this case, the challenge is triggered and privacyIDEA expects a response.
+        If the user now gives the answer expected from the server, the response is accepted and the authentication is successful.
+
+        Multi Challenge is basically a chain of challenges. If, for example, a PIN is stored for a challenge response token, after
+        the first successful challenge, a second one is triggered, in which the PIN is expected as an answer.
+        If the PIN matches, the second challenge is also successful and the user is successfully authenticated.
+
+   Policy Conditions
+        Since privacyIDEA 3.1, :ref:`policy_conditions` allow to define more advanced rules
+        for policy matching, i.e. for determining which policies are valid for a
+        specific request.
+
+        Conditions can be added to a policy via the WebUI. In order for a policy to
+        take effect during the processing of a request, the request has to match not
+        only the ordinary policy attributes (see :ref:`policies`), but also *all*
+        additionally defined conditions that are currently active. If no active
+        conditions are defined, only the ordinary policy attributes are taken into
+        account.
+
+   Application Plugins
+        privacyIDEA comes with application plugins. These are plugins for
```

Comment:
I would say: Plugins are available.

"comes with" could cause false expectations.

Reply:
done in 53bb67399db07edeb0402718201637498de5b8a2.